pkg/streamingpromql/engine_test.go

```diff
@@ -2801,7 +2801,7 @@ func TestCompareVariousMixedMetricsVectorSelectors(t *testing.T) {
 
 	for _, labels := range labelCombinations {
 		labelRegex := strings.Join(labels, "|")
-		for _, function := range []string{"rate", "increase", "changes", "resets", "deriv", "irate", "idelta"} {
```

Comment:
We should not remove deriv from here.

Reply:
👍 Thanks. I removed it initially because I thought it would fix the tests. I needed to do actual fixes instead 😄